.golangci.yaml

```diff
@@ -1,20 +1,32 @@
 linters-settings:
-  lll:
-    line-length: 120
```

Comment:
Why did this get removed?

Reply:
It's [a default setting](https://golangci-lint.run/usage/linters/#lll) for the `lll` plugin 🙂